_posts/en/newsletters/2023-10-25-newsletter.md

```diff
@@ -0,0 +1,588 @@
+  compact block relay] means the transaction data only needs to be broadcast
+  once to a peer.
+
+- [Why are OP_MUL and OP_DIV disabled in Bitcoin?]({{bse}}119785)
+  Antoine Poinsot points out that the `OP_MUL` and `OP_DIV` opcodes were
+  probably disabled, in addition to [other opcodes][github disable opcodes], as
+  a result of the ["1 RETURN"]({{bse}}38037) and [OP_LSHIFT crash][CVE-2010-5137] bugs discovered
+  in the weeks prior.
+
+- [Why are hashSequence and hashPrevouts computed separately?]({{bse}}119832)
+  Pieter Wuille explains that by splitting up to-be-signed transaction hash data
+  into previous outputs and sequences, those hash values can be used once for the whole
+  transaction involving all types of sighashes.
+
+- [Why does Miniscript add an extra size check for hash preimage comparisons?]({{bse}}119892)
+  Antoine Poinsot notes that hash preimages are limited in size in
+  [miniscript][topic miniscript] to avoid non-standard Bitcoin transactions,
+  avoid consensus-invalid cross-chain atomic swaps, and to ensure that witness
+  costs can be accurately calculated.
+
+- [How can the next block fee be less than the mempool purging fee rate?]({{bse}}120015)
+  User Steven references mempool.space dashboards showing a default mempool
+  purging 1.51sat/vb transactions while also indicating an estimated next block
+  containing transactions with 1.49sat/vb. Glozow outlines the likely
+  explanation as a full mempool resulting in the eviction of a transaction that
+  bumped the node's mempool min feerate (by `-incrementalRelayFee`) but left
+  some lower feerate transactions in the mempool that did not need to be
+  evicted in order to stay within the max mempool size.
+
+  She also mentions the asymmetry between [ancestor scoring][waiting for confirmation 2] for block
+  template selection and descendant scoring for mempool eviction as another
+  possible explanation and links to a [cluster mempool][topic cluster
+  mempool]-related [issue][Bitcoin Core #27677] explaining the asymmetry and a
+  potential new approach.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 25.1][] is a maintenance release mainly containing bug
+  fixes.  It is the current recommended version of Bitcoin Core.
+
+- [Bitcoin Core 24.2][] is a maintenance relaese mainly containing bug
+  fixes.  It is recommend for anyone still using 24.0 or 24.1 who is
+  unable or unwilling to upgrade to 25.1 at this time.
+
+<!-- FIXME: bitschmidty to test whether the 26.0 link works before
+publication.  If it does, please remove the "Verified test..." sentence.
+In either case, please also delete this comment.  (Thanks, Mike!) -->
+
+- [Bitcoin Core 26.0][] is a release candidate for the next major
```

Comment:
fyi @bitschmidty seems quite unlikely to have bins up before the newsletter is published.

Reply:
Also:
```suggestion
<!-- FIXME: bitschmidty to test whether the 26.0 link works before
publication.  If it does, please remove the "Verified test..." sentence.
In either case, please also delete this comment.  (Thanks, Mike!) -->

- [Bitcoin Core 26.0rc1][] is a release candidate for the next major
```